src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300px;
+}
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800px;
+}
+
+body {
+  margin-top: 20px;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  color: rgba (61, 78, 97, 0.1);
+  background: #fff;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  font-weight: 800px;
```

Comment:
such `font-weight` must be implemented when the bar is in focus.
```suggestion
```
You already have the right weight in .body

Reply:
Hi Andrii,
Thank you for your remark. I fixed font-size as you said.
But npm-test still almost completely failed.